crates/sui-single-node-benchmark/src/single_node.rs

```diff
@@ -161,12 +159,51 @@ impl SingleValidator {
                     .await;
                 response.signed_effects.into_data()
             }
-            Component::TxnSigning => unreachable!(),
+            Component::TxnSigning | Component::CheckpointExecutor | Component::ExecutionOnly => {
+                unreachable!()
+            }
         };
         assert!(effects.status().is_ok());
         effects
     }
 
+    pub(crate) async fn execute_transaction_in_memory(
```

Comment:
IIUC, this function avoids reading object from rocksdb, but still writes output to rocksdb? Impressive that it already improves throughput this much.

Reply:
No, it doesn't interact with rocksdb at all.

Reply:
Got it, this skips committing effects and other results.